sympy/vector/tests/test_parametricregion.py

```diff
@@ -0,0 +1,50 @@
+from sympy import sin, cos, pi, symbols
+from sympy.vector.coordsysrect import CoordSys3D
+from sympy.vector.parametricregion import ParametricRegion
+
+C = CoordSys3D('C')
+r, theta, phi, a, b, t = symbols('r theta phi a b t')
+
+def test_parametricregion():
+
+    point = ParametricRegion(3, 4)
+    assert point.definition == [3, 4]
+    assert point.parameters == []
+    assert point.bounds == []
+    assert point.system == None
+
+    l = ParametricRegion(C.y, (C.y, -3, 3), system=C)
```

Comment:
Maybe we could have either `ParametricRegion((y,), (y, -3, 3))` or `ParametricRegion(C, (C.y, -3, 3))`.

That is, if the first argument is a tuple, then it's assumed to be a list of variable-parameters. If it is a `CoordSys3D` object, then use its scalars as parameters.

This means that `ParametricRegion` would always take 2 arguments when called. What do you think of this API?

Reply:
> That is, if the first argument is a tuple, then it's assumed to be a list of variable-parameters. If it is a CoordSys3D object, then use its scalars as parameters.

Yes, this API looks simple and easy but
```python
ParametricRegion(C, (C.y, -3, 3))
```
I am not able to understand what this region represents. 
 For me, `ParametricRegion(C.y, (C.y, -3, 3), system=C)` represents the line y=x where y varies from -3 to 3. Does this represent the same region?

Reply:
> For me, `ParametricRegion(C.y, (C.y, -3, 3), system=C)` represents the line y=x where y varies from -3 to 3. Does this represent the same region?

Oh, I see. With `ParametricRegion(C, (C.y, -3, 3))` I think of the line y = -3 and z = 3, with `C.y` being the parameter varying along the x-axis (x-axis because it's in the first position).

Reply:
> line y = -3 and z = 3, with C.y being the parameter varying along the x-axis

I think we can define this region without using the coordinate scalar. How is this different from 
```python
ParametricRegion((t), (t, -3, 3))
```

Reply:
`(t)` is not a tuple, `(t,)` is. Unfortunately Python has some strange syntax.

Reply:
Okay. Then, to make it easier for users, we can put a condition like
```python
if not isinstance(parameter, tuple):
    parameter = tuple(parameter)
```